metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -2189,7 +2189,7 @@ int getTopicEffectiveMinIsr(String topicName) {
         if (minIsrConfig != null) {
             currentMinIsr = Integer.parseInt(minIsrConfig);
         } else {
-            log.warn("Can't find the min isr config for topic: " + topicName + " using default value " + defaultMinIsr);
+            log.debug("Can't find the min isr config for topic: " + topicName + " using default value " + defaultMinIsr);
```

Comment:
This comment is for the `log.warn` message in line 2200.

What exception are you trying to catch exactly? Is it a `NullPointerException`? If so, shouldn't we throw the exception up the stack? Based on the code I would assume that if there is an error it means that the controller is trying to find the replication factor for a topic and doesn't exists, no?

If you decide to keep the log message, lets log the exception instead of logging the exception's `toString`.
```java
log.warn("Can't find the replication factor for topic: " + topicName + " using default value " + replicationFactor, e);
```

Reply:
Make sense, thanks! Removed the error message.